cves/CVE-2013-0861.yml

```diff
@@ -201,8 +220,10 @@ specification:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  answer_note:
-  answer:
+  answer_note: |
+    After looking at the bug reports, commit messages, I was unable to find a 
```

Comment:
Maybe use something like "there were no specification violations mentioned in the bug reports or commit messages" instead of  using "I".

Reply:
recommendation for specification violation entry taken and committed.